### PR TITLE
feat: Add support lending page

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**
Screenshot 2019-07-18 at 08 07 58

### CODE REVIEW

**Comment by jagodapp** [comment]
Looks absolutely amazing 🎉 

The only thing I noticed is that we have a horizontal scrollbar there, probably because of the element in the header?

Screen Shot 2019-07-16 at 11 16 48

File: user/themes/centra/scss/_content.scss [hunk 1]
```diff
@@ -17,8 +17,9 @@
       color: #AAAAAA;
     }
   }
+}
 
-  h1,
```
Comment (kparzecki) [inline user/themes/centra/scss/_content.scss]: Why remove `h1` selector?

File: user/themes/centra/scss/_navigation.scss [hunk 2]
```diff
@@ -192,16 +191,19 @@ $menu-breakpoint: 715px;
...
+    top: 0;
+    opacity: 1;
+    background-color: $black;
+  }
 
-    &.hideNavigation {
-      top: -80px;
-      opacity: 0;
-    }
+  #navigation-wrapper.hideNavigation {
```
Comment (kparzecki) [inline user/themes/centra/scss/_navigation.scss]: `.showNavigation` and `.hideNavigation` can be nested inside `#navigation-wrapper`.

File: user/themes/centra/scss/support-page/_support-search.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    animation: swimming5 10s infinite;
+    animation-direction: alternate;
+    background-size: contain;
+  }
+
+  .navigation-background__smile {
+    display: none;
+  }
+
+  .navigation-background__green-steps {
```
Comment (kparzecki) [inline user/themes/centra/scss/support-page/_support-search.scss]: `.navigation-background__smile` and `.navigation-background__green-steps` have the same css properties.

File: user/themes/centra/templates/partials/navigation_search.html.twig [hunk 4]
```diff
@@ -1,15 +1,31 @@
 {% block navigation_search %}
     <div class="navigation-search" id="search">
         <forma class="navigation-search__form">
-            <input class="navigation-search__input" type="search" id="site-search" name="q" placeholder="Search Centra Developers" onkeydown = "if (event.keyCode == 13) document.getElementById('navigation-search-button').click()">
+            <input
+              class="navigation-search__input"
+              type="search" id="site-search"
+              name="q"
+              placeholder='{{ grav.theme.config.site_version is same as("api-doc") ? "Search Centra Developers" : "Search Centra Support" }}'
+              onkeydown = "if (event.keyCode == 13) document.getElementById('navigation-search-button').click()"
```
Comment (kparzecki) [inline user/themes/centra/templates/partials/navigation_search.html.twig]: `keyCode` is deprecated. Please use `key` property instead.

File: user/themes/centra/templates/partials/support-landing-page/support-landing-page-showroom.html.twig [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{% from 'macros/docs-articles.html.twig' import docs_articles -%}
+
+<section class="landing-page-section docs support-documentation support-documentation_showroom" id="docs-section">
+  <div class="section-wrapper docs-section">
+    <h3 class="name-section" data-aos="lift-name-section">Centra Showroom</h3>
+    <h1 class="section-title">Showroom documentation & tutorials</h1>
```
Comment (kparzecki) [inline user/themes/centra/templates/partials/support-landing-page/support-landing-page-showroom.html.twig]: Please use html entity for ampersand symbol (`&amp;`).

File: user/themes/centra/templates/partials/support-landing-page/support-langing-page-newsletter-updates.html.twig [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<section class="landing-page-section">
+  <div class="section-wrapper section-newsletter-updates">
+    <div class="support-section-list" data-aos="lift-docs-cards">
+      <h1 class="section-title">Newsletter archive</h1>
+      <a>View all</a>
+      <ul>
+        {% set newsletter = page.find('/newsletter/newsletter/2019').children.order('date', 'desc') %}
+        {% for p in newsletter if p != page %}
+          <li><a href="{{p.url}}">{{ p.title }}</a></li>
```
Comment (kparzecki) [inline user/themes/centra/templates/partials/support-landing-page/support-langing-page-newsletter-updates.html.twig]: Please keep the spacing in twig statements consistent across the project.

**Comment by kparzecki** [comment]
I really don't like the usage of `grav.theme.config.site_version` to decide how the site should look like. It would be better to use theme inheritance feature from grav, which would provide much more understandable and cleaner solution to the problem.
https://learn.getgrav.org/16/themes/customization#theme-inheritance

File: user/themes/centra/scss/_content.scss (hunk 1 above)
Comment (NataliiaKononikhina) [reply user/themes/centra/scss/_content.scss]: Because it has another font-weight

**Comment by kparzecki** [comment]
image

Animated icons/images are overflowing page container.
Also on smaller screens the logo looks strangely small.

**Pull request merged**